app/views/works/_license_text.html.erb

```diff
@@ -0,0 +1,50 @@
+<p>
+In order for Princeton University's Princeton Data Commons to reproduce, translate, and distribute
+your submission for worldwide public access your agreement to the following terms is necessary.
+Please take a moment to read the terms of this license.
+</p>
```

Comment:
As I understand it, this page will be linked from PDC Discovery, where it will describe the terms under which access is granted to this object. In that context, I don't think "your submission" makes sense.

I would leave this paragraph in review.html.erb, and perhaps replace it here with a different paragraph addressed to the end-user.